framework/mesh/unpartitioned_mesh/unpartitioned_mesh.cc

```diff
@@ -625,8 +622,8 @@ UnpartitionedMesh::CopyUGridCellsAndPoints(vtkUnstructuredGrid& ugrid,
 
   if (has_global_ids)
   {
-    std::vector<CellPtr> cells(total_cell_count, nullptr);
-    std::vector<Vec3Ptr> vertices(total_point_count, nullptr);
+    std::vector<LightWeightCell*> cells(total_cell_count, nullptr);
+    std::vector<Vector3*> vertices(total_point_count, nullptr);
```

Comment:
@andrsd  are those raw pointers? what do you think of seeing a few of these int he code?

Reply:
Yes, they are. I personally have nothing against  raw pointers. I learned coding before smart pointers were a thing in C++, so I am conditioned on memory allocation/deallocation. But, in more complicated codes it can easily happen that memory is not deallocated and memory leaks can occur. Then, you can have things like double free, etc.  

So, if you are thinking of solely using smart pointers (for consistency) and prevent memory related issues, then it needs to be a buy-in from all (or most) developers. And mixing raw and smart pointers should not be a thing, I think - developers will be always thinking about "was this a raw pointer or a smart pointer"...

Reply:
so, there should be no time penalty between raw and smart pointers? if so, should we remove all raw pointers?

Reply:
I would vote to remove raw pointers for objects

Reply:
I think we should eliminate raw pointers wherever we can. We're going to get more and more students working in the code that probably won't be very familiar with handling raw pointers. Eliminating them wherever possible will save us a lot of headaches in the future.

Reply:
so there is a consensus to remove raw pointers (or most of them). The only question that remains is whether this happens in this PR or if an issue gets created for a subsequent PR. Thanks!

Reply:
As a practice, I think the scope of PRs should not be expanded unless absolutely necessary or the change is very small. If the only raw pointer in the code is the one here, I would change this PR, if there are more, I would create an issue and address in a new PR.

Reply:
I opened #95 for raw pointers elimination.